### PR TITLE
dunst: add dunst, a do-not-disturb button for muting notifications

### PR DESCRIPTION
**Pull request by jcinis**
A blocklet that adds a very clean and simple do-not-disturb button for silencing dunst notifications.  The behavior is modeled after Apple's "Do Not Disturb" toggle, however instead of using a moon icon it uses a bell.

### CODE REVIEW

**Comment by da-edra** [comment]
@jcinis I was looking for something like this, thank you!

BTW, I think you forgot to add your blocklet to the README.md.

**Comment by jcinis** [comment]
> @jcinis I was looking for something like this, thank you!
> 
> BTW, I think you forgot to add your blocklet to the README.md.

Thanks so much @da-edra!

**Comment by da-edra** [comment]
Hello @jcinis I know it's been a while since you opened this PR, I'd like to integrate it into a fork we're creating to keep the [i3blocks-contrib](https://github.com/da-edra/i3blocks-contrib) repository updated.

You PR looks pretty good! It is a nice and useful block, in case you're still using i3blocks and interested in it getting merged could you please fix the following errors pylint found?

```bash
************* Module dunst
dunst.py:22:4: W1510: Using subprocess.run without explicitly set `check` is not recommended. (subprocess-run-check)
dunst.py:31:4: W1510: Using subprocess.run without explicitly set `check` is not recommended. (subprocess-run-check)

-----------------------------------
Your code has been rated at 9.26/10
```

Thanks in advance!

**Comment by da-edra** [comment]
Hello @jcinis , I now have write access to this repository and I'll be maintaining it.

Your branch looks good to me! I'll merge it as soon as the pylinting warnings are corrected.

Thanks for sharing!

File: dunst/dunst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+__license__ = "MIT"
+__version__ = "1.0.0"
+
+import os
+import subprocess
+import json
+
+def mute_on():
+    '''Turns off dunst notifications'''
+    subprocess.run(["notify-send","DUNST_COMMAND_PAUSE"])
```
Comment (da-edra) [inline dunst/dunst]: Using `subprocess.run` without explicitly set `check` is not recommended.

File: dunst/dunst [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    return {
+        "full_text":"<span font='Font Awesome 5 Free Solid' color='#BE616E'>\uf1f6</span>",
+        "DUNST_MUTE":"on"
+    }
+
+def mute_off():
+    '''Turns back on dunst notifications'''
+    subprocess.run(["notify-send","DUNST_COMMAND_RESUME"])
```
Comment (da-edra) [inline dunst/dunst]: Using `subprocess.run` without explicitly set `check` is not recommended.

File: dunst/dunst (hunk 1 above)
Comment (jcinis) [reply dunst/dunst]: Added a new commit with check explicitly set.

File: dunst/dunst (hunk 2 above)
Comment (jcinis) [reply dunst/dunst]: Added a new commit with check explicitly set.

**Comment by jcinis** [comment]
The latest commit should now pass pylint.  Thanks for the feedback; I hope the blocklet is useful to others. :boom:

**Pull request merged**